pootle/checks.py

```diff
@@ -262,6 +262,73 @@ def check_settings(app_configs=None, **kwargs):
                     id="pootle.W015",
                 ))
 
+    if settings.POOTLE_TM_SERVER:
+        tm_indexes = []
+
+        for server in settings.POOTLE_TM_SERVER:
```

Comment:
I can't help but feel that there has to be a better way to check these configurations.